resources/assets/icons/status/info.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-miterlimit="10" d="M10 9.2v5"/><path stroke-linecap="round" stroke-linejoin="round" d="M10 5.8h0"/></g></svg>
```

Comment:
identical to
![image](https://user-images.githubusercontent.com/35610748/128482376-a1133217-a83b-485b-9b28-d52581714c16.png)